doc/codi.txt

```diff
@@ -214,6 +214,12 @@ g:codi#interpreters
              Configuration is deeply applied over the default interpreters.
              Default value is {}.
 
+                                                    *g:codi#environment_flags*
+g:codi#environment_flags
```

Comment:
I wonder if it makes sense to use a format string to specify a custom command wrapper, rather than explicitly have it be environment flags.

Reply:
True, seems sensible. Another option is to simply always use the default value as a non-empty `inputrc` can only break things but does little good in codi's parsing of the repl output.

Reply:
I can imagine that there might be some user somewhere that wants their inputrc read for whatever reason 😛 Let's proceed with this configuration for now, and if we want to change/add anything we can do so in a follow-up PR!

Reply:
Regarding a custom command wrapper, this would make perfect sense as it

- is more flexible, and
-  also permits giving a priority to the background job  by `nice` say.